translations/base-pt-BR.yaml

```
@@ -527,31 +527,31 @@ buildings:
         levelShortcut: LVL
     wire:
         default:
-            name: Energy Wire
-            description: Allows you to transport energy.
+            name: &wire Cabo de Energia
+            description: Permite transportar energia.
     advanced_processor:
         default:
-            name: Color Inverter
-            description: Accepts a color or shape and inverts it.
+            name: Inversor de Cor
+            description: Aceita uma cor ou forma e a inverte.
     energy_generator:
-        deliver: Deliver
-        toGenerateEnergy: For
+        deliver: Entregar
+        toGenerateEnergy: Para
         default:
-            name: Energy Generator
-            description: Generates energy by consuming shapes.
+            name: Gerador de Energia
```

The `&energy_generator` anchor is missing here. It is aliased from line 810.
```suggestion
            name: &energy_generator Gerador de Energia
```